packages/@glimmer/interfaces/lib/compile/wire-format.d.ts

```diff
@@ -103,8 +77,7 @@ export const enum ExpressionContext {
   // An `Append` is a single identifier that is contained inside a curly (either in a
   // content curly or an attribute curly)
   AppendSingleId = 'AppendSingleId',
-  // An `Expression` is evaluated into a value (e.g. `person.name` in `(call person.name)`
-  // or `person.name` in `@name={{person.name}}`). This represents a syntactic position
+  // An `Expression` is evaluated into a value (e.g. `person.name` in `(call person.name)` // or `person.name` in `@name={{person.name}}`). This represents a syntactic position
```

Comment:
This is a bit odd, to have two sets of `//` on a line

Reply:
this was done with format on save.